lib/webdriver.py

```diff
@@ -149,7 +149,8 @@ def _get_driver(self) -> Driver:
         logger.debug("Loading Southwest home page (this may take a moment)")
         driver.open(BASE_URL)
         self._take_debug_screenshot(driver, "after_page_load.png")
-        driver.js_click("(//div[@data-qa='placement-link'])[2]")
+        driver.wait_for_element("//*[@alt='Check in banner']", timeout=30)
```

Comment:
Hm, I wonder if we should handle this function timing out. If it does though, I don’t see a good approach to handling this anyway except waiting for 24 hours to check again.